pkg/new-ui/v1beta1/util.go

```diff
@@ -94,8 +95,31 @@ func (k *KatibUIHandler) getTrialTemplatesViewList() ([]TrialTemplatesDataView,
 			return nil, err
 		}
 
+		// Iterate over the trialTemplatesConfigMapList from all namespaces and filter out the
+		// configmaps that belong to namespaces other than kubeflow and the ones that the user has
+		// access privileges.
+		var newTrialTemplatesConfigMapList []apiv1.ConfigMap
+		for _, cmap := range trialTemplatesConfigMapList.Items {
+			if ns == "kubeflow" {
```

Comment:
Can you use this `const` variable please for `kubeflow` namespace: https://github.com/kubeflow/katib/blob/master/pkg/controller.v1beta1/consts/const.go#L175 ?